### DISCUSSION TITLE
Alembic not creating index with BigInteger sa_column type

### DISCUSSION THREAD

**Discussion by robcxyz** [fastapi/sqlmodel#554] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from sqlalchemy import BigInteger
from sqlmodel import Field, SQLModel, Column

class Thing(SQLModel, table=True):
    id: int = Field(None, primary_key=True)
    not_indexed_in_alembic: int = Field(None, index=True, sa_column=Column(BigInteger()))
    is_indexed_in_alembic: int = Field(None, index=True)
```

### Description

Please excuse me if this is not a sqlmodel question but more to do with alembic, but figured I'd post it here first as it relates to issue #191, specifically [this comment](https://github.com/tiangolo/sqlmodel/issues/191#issuecomment-1001838728). 

When I am using a custom sqlalchemy type `BigInteger`, Alembic is not creating an index for me on that column. 

To reproduce:
1. Import the table into alembic's env.py
2. Run `alembic revision --autogenerate -m "testing" &&  alembic upgrade head`
3. No index is created for the BigInteger column

Also checked this with older versions of sqlmodel and still got same issue. 

### Operating System

Linux

### Operating System Details

Ubuntu 

### SQLModel Version

v0.8.0

### Python Version

3.10

### Additional Context

Alembic is latest version (1.9.3)

**Comment by jochman** [comment]
joining, same problem

**Comment by YuriiMotov** [comment, marked as answer]
When you use `sa_***` parameters, it hides other column-related parameters like `index`, `unique`, `primary_key`, ..
You need to pass these parameters to `Column` instead:

```py
    not_indexed_in_alembic: int = Field(
        None,
        # index=True, <- this will not work
        sa_column=Column(BigInteger(), index=True)  # <- pass column-related parameters here
    )
```
